layouts/partials/navigation.html

```diff
@@ -22,11 +22,15 @@
 </ul>
 
 <ul class="subscription">
+  {{ $rssLink := "/index.xml" | absURL }}
+  {{ with .OutputFormats.Get "RSS" }}
```

Comment:
This is required since I have a different rss url with this config:

```toml
[outputFormats]
[outputFormats.RSS]
mediatype = "application/rss"
baseName = "atom"
```